whisper_medusa/models/model.py

```diff
@@ -1458,3 +1670,65 @@ def generate(
 
         else:
             raise NotImplementedError("Longform generation is not supported yet")
+
+    def freeze_model_parts(self, parts_to_freeze):
+        self.whisper_model.freeze_model_parts(parts_to_freeze)
+
+    def set_output_whisper_original(self):
```

Comment:
make private it won't be called from outside.

Reply:
but we call it from the trainer.py

Reply:
the freeze func was included by mistake I was referring to `set_output_whisper_original`

Reply:
done